### PR TITLE
framework/media : Add error handling statement

### PR DESCRIPTION
**Pull request by dev-hue**
Add error handling about AUDIO_MANAGER_DEVICE_NOT_SUPPORT in setPlayerVolume()

### CODE REVIEW

File: framework/src/media/MediaPlayerImpl.cpp [hunk 1]
```diff
@@ -428,7 +428,11 @@ void MediaPlayerImpl::setPlayerVolume(uint8_t vol, player_result_t &ret)
 	audio_manager_result_t result = set_output_audio_volume(vol);
 	if (result != AUDIO_MANAGER_SUCCESS) {
 		meddbg("set_input_audio_volume failed vol : %d ret : %d\n", vol, result);
-		ret = PLAYER_ERROR_INTERNAL_OPERATION_FAILED;
+		if (result == AUDIO_MANAGER_DEVICE_NOT_SUPPORT) {
```
Comment (Taejun-Kwon) [inline framework/src/media/MediaPlayerImpl.cpp]: please check utc too.

**Comment by seinfra** [comment]
Target : [6898dd20a3428fb7a8b06b4191688f65153305a4] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6898dd20a3428fb7a8b06b4191688f65153305a4] - Code Rule Check (C++) OK.

**Comment by dev-hue** [comment]
Close this Pull Request because of new PR contains additional modified.

**Pull request closed without merging**